backend-nest/tslint.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
We will have to replace TSLINT with ESLINT ... https://medium.com/palantir/tslint-in-2019-1a144c2317a9